### ISSUE TITLE
Crash:  longjmp causes uninitialized stack frame

### ISSUE THREAD

**Issue by peperunas** [Vector35/binaryninja-api#1009]
Binary Ninja crashes with the following error when opening the challenge `Replace.exe` from [reversing.kr](http://reversing.kr).

```
 longjmp causes uninitialized stack frame ***: binaryninja terminated
```

# How to reproduce
Just open the binary and wait for ~10 minutes.

**Comment by GermaniumSystem** [comment]
I have been experiencing the same issue. After 5 minutes, Binary Ninja crashes with the following trace:
```*** longjmp causes uninitialized stack frame ***: ./binaryninja terminated
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7faa033217e5]
/lib/x86_64-linux-gnu/libc.so.6(__fortify_fail+0x5c)[0x7faa033c315c]
/lib/x86_64-linux-gnu/libc.so.6(+0x11905d)[0x7faa033c305d]
/lib/x86_64-linux-gnu/libc.so.6(__longjmp_chk+0x29)[0x7faa033c2fb9]
/home/bogon/Downloads/binaryninja/libcurl.so.4(+0xa135)[0x7faa03050135]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x11390)[0x7faa03f26390]
/lib/x86_64-linux-gnu/libc.so.6(__poll+0x2d)[0x7faa033a574d]
/lib/x86_64-linux-gnu/libglib-2.0.so.0(+0x4a38c)[0x7faa002b838c]
/lib/x86_64-linux-gnu/libglib-2.0.so.0(g_main_context_iteration+0x2c)[0x7faa002b849c]
/home/bogon/Downloads/binaryninja/libQt5Core.so.5(_ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE+0x5f)[0x7faa04404baf]
/home/bogon/Downloads/binaryninja/qt/platforms/../../libQt5XcbQpa.so.5(+0xd89a1)[0x7fa9fcb5c9a1]
/home/bogon/Downloads/binaryninja/libQt5Core.so.5(_ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE+0xea)[0x7faa043af26a]
/home/bogon/Downloads/binaryninja/libQt5Core.so.5(_ZN16QCoreApplication4execEv+0x84)[0x7faa043b7da4]
./binaryninja[0x43c7d4]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0)[0x7faa032ca830]
./binaryninja[0x445740]
```

This appears to be related to issue #962 as the workaround listed there got my installation working for more than 5 minutes.

**Comment by bpotchik** [comment]
Fixed in build 1.1.1229-dev. Proxy support has been added as well.
